aikau/src/main/resources/alfresco/util/hashUtils.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Utility object for hash-related utilities. Note that this is not a Class, and so does
+ * not need to be instantiated before use.
+ *
+ * @module alfresco/util/hashUtils
```

Comment:
I like these utils and the pattern they're defined with.

Is it worth adding a method to return the hash as a string. That's used in a few places in the code so would enable these utils to replace a direct call to dojo/hash (direct library calls always make me wary.)